Automattic/wp-calypso
# Domains: Add new material icons we'd like to use for the new domain status screens

I've added most of the icons we're going to use. We might add some more later but for now let's go with those

#### Changes proposed in this Pull Request

* Added a bunch of material icons we're going to use in the new domain status screens

#### Testing instructions

* I guess you can go to the playground and paste the following:
```
<Main>
<MaterialIcon icon="cached" />
<MaterialIcon icon="chrome_reader_mode" />
<MaterialIcon icon="delete" />
<MaterialIcon icon="info" />
<MaterialIcon icon="language" />
<MaterialIcon icon="email" />
<MaterialIcon icon="cloud_upload" />
</Main>
```
You should see 7 icons
